tests/integration_core.rs

```diff
@@ -0,0 +1,58 @@
+#[allow(dead_code)]
+mod common;
+
+use reqwest;
+use std::error::Error;
+use wascc_host::WasccHost;
+
+#[test]
+fn stock_host() -> Result<(), Box<dyn Error>> {
+    let host = common::gen_stock_host()?;
+    assert_eq!(2, host.actors().len());
+    if let Some(ref claims) =
+        host.claims_for_actor("MB4OLDIC3TCZ4Q4TGGOVAZC43VXFE2JQVRAXQMQFXUCREOOFEKOKZTY2")
+    {
+        let md = claims.metadata.as_ref().unwrap();
+        assert!(md
+            .caps
+            .as_ref()
+            .unwrap()
+            .contains(&"wascc:http_server".to_string()));
+    }
+
+    std::thread::sleep(::std::time::Duration::from_millis(100));
+
+    let resp = reqwest::blocking::get("http://localhost:8081")?;
+    assert!(resp.status().is_success());
+    assert_eq!(resp.text()?,
+        "{\"method\":\"GET\",\"path\":\"/\",\"query_string\":\"\",\"headers\":{\"accept\":\"*/*\",\"host\":\"localhost:8081\"},\"body\":[]}"
+    );
+    host.shutdown()?;
+    std::thread::sleep(::std::time::Duration::from_millis(100));
```

Comment:
Are these sleeps required here?

Reply:
No, the sleeps after shutdown are not necessary. I originally had them because my tests were sharing the same HTTP port, but it was far more easier and reliable to give each test its own HTTP port.